pkg/skaffold/kubectl/exec_windows.go

```diff
@@ -42,54 +45,58 @@ func CommandContext(ctx context.Context, name string, arg ...string) *Cmd {
 }
 
 // Start starts the specified command in a job object but does not wait for it to complete
-func (c *Cmd) Start() (err error) {
-	var handle windows.Handle
-	handle, err = windows.CreateJobObject(nil, nil)
+func (c *Cmd) Start() error {
+	handle, err := windows.CreateJobObject(nil, nil)
```

Comment:
Is this `handle` a lightweight object or does it consume system resources?  Does it need to be cleaned up with `windows.CloseHandle()` before we return with an error?